ecommerce/auth/router.py

```diff
@@ -1,30 +1,45 @@
-from fastapi import APIRouter, Depends, HTTPException, status
-from .schemas import UserOut, UserAuth
+from fastapi.security import OAuth2PasswordBearer, OAuth2PasswordRequestForm
+from fastapi import HTTPException, status, Depends, APIRouter
 
 
-from ecommerce.main import get_db
-from ecommerce.User.model import Users
-from ecommerce.main import create_user
+from app.database import get_db
+from user.model import Users
+from user.utils import create_user, read_user, get_current_user
+from user.password import verify_password
+from auth.schemas import UserOut, UserAuth, TokenOut
+from auth.json_token import create_access_token, create_refresh_token
 
-router = APIRouter(tags=["auth"])
+router = APIRouter(tags=["Auth"])

```

Comment:
```suggestion
router = APIRouter(tags=["auth"])

```
Previous was correct, do HTTP paths always `lower-case-with-hyphens`